docs/docs/installation/pypi.mdx

```diff
@@ -100,24 +98,18 @@ We highly recommend installing Superset inside of a virtual environment. Python
 pip install virtualenv
 ```
 
-You can create and activate a virtual environment using:
-
+First create a directory for Superset:
 ```
-# virtualenv is shipped in Python 3.6+ as venv instead of pyvenv.
-# See https://docs.python.org/3.6/library/venv.html
-python3 -m venv venv
-. venv/bin/activate
+mkdir superset
 ```
 
-Or with pyenv-virtualenv:
```

Comment:
I think we still want to keep `pyenv` documentation around, i.e., developers may want to switch which Python environment they want to use. In essence `pyenv` is a precursor to setting up Superset.

Reply:
Again I'm new to this but my newbie perspective was: the docs say:
> You can create and activate a virtual environment using: [_virtual env instructions_] ... Or with pyenv-virtualenv:
I was unclear, which should I use?  And when I read a little about it, it seemed like the `venv` was the most widely-used and so I eliminated an alternative.

If we should include both, can someone add a note to help a user decide which to use?

Reply:
@john-bodley circling back to this -- are you good with my changes or is there another committer you think could weigh in?  Or if nothing else I can revert these parts, I'd rather just get the bulk of this merged.